content/doc/book/index.html.haml

```diff
@@ -20,6 +20,6 @@ title: "Jenkins Handbook"
               %p{:style => 'font-size: 14px;'}
                 = section.summary
 
-.page-link
+.page-link{:style => 'font-size: 24px;'}
```

Comment:
When I compare the "before" and "after", I think that this change of text size will confuse readers, since it is the same size as the contents list that immediately precedes it.

## Before (current site)

![before-text-smaller-than-contents-list](https://github.com/jenkins-infra/jenkins.io/assets/156685/8802294a-8145-4bc9-a889-193a2024f567)

## After (this pull request)

![after-text-looks-same-as-contents-list](https://github.com/jenkins-infra/jenkins.io/assets/156685/83915cf1-4295-4fb0-aa27-78d7d4ad2dbc)

```suggestion
.page-link
```